doc/antlr-project-testing.md

```diff
@@ -190,68 +149,55 @@ Use this to run tests in parallel:
 
 ```bash
 $ export MAVEN_OPTS="-Xmx1G"
-$ mvn -Dparallel=methods -DthreadCount=4 test
+$ mvn -Dparallel=classes -DthreadCount=4 test
```

Comment:
Why classes? It looks like all methods are isolated and they can be and should be tested in parallel. Also, the change is unrelated to the header of the pull request, I recommend dropping this change.

Reply:
I ran into a problem on my M1 Mac for Go; it was creating hundreds of swift-frontend threads and I almost had to reboot the system because I couldn't even get a prompt...

Reply:
It's weird because there is a limit on threads count (4). Have switching to `classes` resolved the problem completely?